.github/workflows/formatting.yml

```diff
@@ -3,9 +3,9 @@ name: Formatting
 
 on:
   push:
-    branches: [ main, dev, 2.x ]
+    branches: [ main, dev, amd-mainline, amd-staging, 2.x ]
   pull_request:
-    branches: [ main, dev, 2.x ]
+    branches: [ main, dev, amd-mainline, amd-staging, 2.x ]
```

Comment:
Unrelated, but we can remove the stale reference to `2.x` here

Reply:
Will do